rules.adoc

```diff
@@ -29,7 +29,7 @@ by the RoboCupJunior Soccer Committee. The English version of these
 rules has priority over any translations.
 
 Teams are advised to check the RoboCupJunior Soccer site
-https://junior.forum.robocup.org/ for OC (Organizational Committee) procedures
+https://junior.forum.robocup.org/ for OC (Organization Committee) procedures
```

Comment:
Do we want to rename what I put as soccer league committee 2022 at the top to just Organization Committee as well for consistency?

Reply:
It's a good question but I think we can do that once the concept of a Committee is a bit more clear (i.e. not now :slightly_smiling_face: )